PWGEM/PhotonMeson/TableProducer/createPCM.cxx

```diff
@@ -239,20 +240,58 @@ struct createPCM {
 
       auto negTracks_coll = negTracks->sliceByCached(o2::aod::track::collisionId, collision.globalIndex());
       auto posTracks_coll = posTracks->sliceByCached(o2::aod::track::collisionId, collision.globalIndex());
-      // LOGF(info, "collision.globalIndex() = %ld , negTracks_coll1 = %ld , posTracks_coll1 = %ld , negTracks_coll2 = %ld , posTracks_coll2 = %ld , negTracks_coll3 = %ld , posTracks_coll3 = %ld",
-      // collision.globalIndex(), negTracks_coll1.size(), posTracks_coll1.size() , negTracks_coll2.size(), posTracks_coll2.size() , negTracks_coll3.size(), posTracks_coll3.size() );
-
-      // Partition<MyFilteredTracks> groupPos = o2::aod::track::signed1Pt > 0.f && nabs(o2::aod::track::collisionId - collision.globalIndex()) < 1.5f;
-      // Partition<MyFilteredTracks> groupEle = o2::aod::track::signed1Pt < 0.f && nabs(o2::aod::track::collisionId - collision.globalIndex()) < 1.5f;
-      // groupPos.bindTable(tracks); //bindTable is too slow
-      // groupEle.bindTable(tracks); //bindTable is too slow
-      // LOGF(info, "collision.globalIndex() = %ld , negTracks = %ld , posTracks = %ld", collision.globalIndex(), groupEle.size(), groupPos.size());
 
       for (auto& [ele, pos] : combinations(CombinationsFullIndexPolicy(negTracks_coll, posTracks_coll))) {
         fillV0Table(collision, ele, pos);
       }
     } // end of collision loop
   }   // end of process
+  PROCESS_SWITCH(createPCM, processSA, "create V0s with stand-alone way", true);
+
+  Preslice<aod::TrackAssoc> trackIndicesPerCollision = aod::track_association::collisionId;
+  void processTrkCollAsso(aod::TrackAssoc const& trackIndices, FullTracksExt const& tracks, aod::Collisions const& collisions, aod::BCsWithTimestamps const&)
```

Comment:
is this process function not looping over anything automatically? Just to be sure, there is no double loop over aod::TrackAssoc ^^"

Reply:
no.

Reply:
okay good!